railties/test/generators/plugin_generator_test.rb

```diff
@@ -226,6 +226,37 @@ def test_skip_javascript
     end
   end
 
+  def test_skip_action_mailer_with_mountable
+    run_generator [destination_root, "--mountable", "--skip-action-mailer"]
+    assert_no_directory "app/mailers"
+  end
+
+  def test_skip_action_mailer_with_api_and_mountable
+    run_generator [destination_root, "--api", "--mountable", "--skip-action-mailer"]
+    assert_no_directory "app/mailers"
+  end
+
+  def test_skip_action_mailer_with_full
+    run_generator [destination_root, "--full", "--skip-action-mailer"]
+    assert_no_directory "app/mailers"
+  end
+
+  def test_skip_active_job_with_mountable
+    run_generator [destination_root, "--mountable", "--skip-active-job"]
+    assert_no_directory "app/jobs"
+  end
+
+  def test_skip_active_job_with_api_and_mountable
+    run_generator [destination_root, "--api", "--mountable", "--skip-active-job"]
+    assert_no_directory "app/jobs"
+  end
+
+  def test_skip_active_job_with_full
+    run_generator [destination_root, "--full", "--skip-active-job"]
+    assert_no_directory "app/jobs"
+  end
```

Comment:
I was wondering if it'll be better to just combine the tests for `--skip-active-job` and `--skip-action-mailer` together. No need to run the generator more times than necessary.

Reply:
make sense 👍